### PR TITLE
Add save schedule lambda

### PR DESCRIPTION
**Pull request by dantebarbieri**
This lambda is invoked when the "Save Schedule" button is pressed. It takes as input the object returned from the postSchedulePeerTeachers lambda and saves the results into the database. It also attempts to return error codes aside from 200 on failure with the possibility of some useful information in the response body.

### CODE REVIEW

File: lambda/postSavedSchedule/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    })
+    
+    dbQuery = `INSERT INTO section_assignment
+                VALUES
+                `;
+    
+    let paramIndex = 1;
+    for (let sectionId of Object.keys(scheduledObj)) {
+        for (let personId of scheduledObj[sectionId]) {
+            dbQuery += `($${paramIndex++}, $${paramIndex++})`;
```
Comment (benbeauchamp7) [inline lambda/postSavedSchedule/index.js]: These ++'s making me a little uncomfy, I know that it's fine though

**Review by benbeauchamp7 — APPROVED** [review]
Short and sweet, looks good to me but won't know for sure until we start hitting it with data

File: lambda/postSavedSchedule/index.js (hunk 1 above)
Comment (Shane-McGookey) [inline lambda/postSavedSchedule/index.js]: This won't work without commas at the end of each set of values. Some logic will be needed to always add a comma except the last line.

File: lambda/postSavedSchedule/index.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            params.push(personId);
+            params.push(sectionId);
+        }
+    }
+    
+    let dbRows = await helper_functions.queryDB(dbQuery, params).catch(err => {
+        response.statusCode = 500;
+        response.body = JSON.stringify({err});
```
Comment (Shane-McGookey) [inline lambda/postSavedSchedule/index.js]: Would put a generic error message here; we don't want the user to directly see our error logs.

**Review by Shane-McGookey — CHANGES_REQUESTED** [review]
Few changes recommended, otherwise looks good.

File: lambda/postSavedSchedule/index.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    }
+
+    dbQuery += dbParams.join(', ');
+    
+    let dbRows = await helper_functions.queryDB(dbQuery, params).catch(err => {
+        response.statusCode = 500;
+        response.body = JSON.stringify({err: "Failed to save the new schedule."});
+    });
+    
+    console.log(dbRows);
```
Comment (Shane-McGookey) [inline lambda/postSavedSchedule/index.js]: Would remove this.

**Review by Shane-McGookey — CHANGES_REQUESTED** [review]
One small change, otherwise looks good.

**Pull request merged**